### PR TITLE
Refactor spender

### PR DESCRIPTION
**Pull request by charlesjhongc**
- Share `Ownable.sol`
- Move events in interface

### CODE REVIEW

File: contracts/interfaces/ISpender.sol [hunk 1]
```diff
@@ -2,6 +2,17 @@
 pragma solidity >=0.7.0;
 
 interface ISpender {
+    // System events
+    event TimeLockActivated(uint256 activatedTimeStamp);
+    // Operator events
+    event SetPendingOperator(address pendingOperator);
```
Comment (NIC619) [inline contracts/interfaces/ISpender.sol:8]:
```suggestion
    // Owner events
```

**Review by changwu-tw — DISMISSED** [review]

**Review by pilagod — DISMISSED** [review]

File: contracts/interfaces/ISpender.sol (hunk 1 above)
Comment (charlesjhongc) [reply contracts/interfaces/ISpender.sol]: Fixed in d7008e9

Comment (NIC619) [reply contracts/interfaces/ISpender.sol]: `SetPendingOperator` event is not used anymore?

Comment (charlesjhongc) [reply contracts/interfaces/ISpender.sol]: Oh right, fixed in 1aa8b3a.

**Review by NIC619 — APPROVED** [review]

**Review by pilagod — APPROVED** [review]

**Pull request merged**